packages/dart_frog_cli/e2e/test/dev_test.dart

```diff
@@ -47,4 +47,80 @@ void main() {
       expect(response.body, 'Route not found');
     });
   });
+
+  group('dart_frog dev when ran multiple times', () {
+    const projectName1 = 'example1';
+    const projectName2 = 'example2';
+
+    final tempDirectory = Directory.systemTemp.createTempSync();
+
+    Process? process1;
+    Process? process2;
+
+    setUp(() async {
+      await dartFrogCreate(projectName: projectName1, directory: tempDirectory);
+      await dartFrogCreate(projectName: projectName2, directory: tempDirectory);
+    });
+
+    tearDown(() async {
+      if (process1 != null) {
+        killDartFrogServer(process1!.pid).ignore();
+      }
+
+      if (process2 != null) {
+        killDartFrogServer(process2!.pid).ignore();
+      }
+    });
+
+    tearDownAll(() {
+      tempDirectory.delete(recursive: true).ignore();
```

Comment:
```suggestion
      tempDirectory.deleteSync(recursive: true);
```

(https://github.com/VeryGoodOpenSource/dart_frog/pull/599#discussion_r1210111489)

Reply:
I did that first. It throws an error that directory does not exist. so I wrote it like the other test group.

Reply:
Does changing `setUp` (where the creation of the directories is done) to `setUpAll` make it work?

Reply:
It did ❤️.

Reply:
AAAAAnd failed on windows. I reverted it back.